lib/AsyncStorage.js

```diff
@@ -359,6 +359,24 @@ const AsyncStorage = {
       });
     });
   },
+
+  /**
+   * Call this set the storage location.
+   * @param storageLocation The storage location. Can be 'documents' or 'applicationSupport'.
+   *
+   * **Note**: changing this only sets the location for future actions and does not migrate
+   * the store from the previous location.
+   *
+   * @platform ios
+   */
+  setStorageLocationIOS: function(storageLocation: string) {
+    RCTAsyncStorage.setStorageLocation(storageLocation);
+  },
+
+  StorageLocationIOS: {
+    documents: 'documents',
```

Comment:
I think we could [export constants to be used in JS](https://facebook.github.io/react-native/docs/native-modules-ios#exporting-constants), what do you think?

Reply:
oh yeah that's a great shout! will do that